src/main/java/com/codejoust/main/service/RoomService.java

```diff
@@ -363,8 +376,8 @@ public RoomDto setSpectator(String roomId, SetSpectatorRequest request) {
             throw new ApiException(RoomError.NOT_FOUND);
         }
 
-        // Return error if user changes status to non-spectator in active game.
-        if (room.getActive() && !request.getSpectator()) {
+        // Return error if user changes status in active game.
+        if (room.getActive()) {
             throw new ApiException(RoomError.ACTIVE_GAME);
```

Comment:
Just to confirm, this is just to disable it completely and making things easier for us to manage right?

Reply:
Yes, that's exactly right 👍